### ISSUE TITLE
Not getting portal vault hot potato messages

### ISSUE THREAD

**Issue by jejorda2** [crawl/crawl#4054]
https://cbro.berotato.org/saves/jejorda2-crawl-git-660b845a87-240926-1642.tar.bz2

I spent about 600 turns walking about D:6 looking for the sewer entrance, and I never got a "you near a very distant drain" message.  Those worked yesterday.  I think I was searching long enough to get two or three messages.

Online console, cbr2, 0.33-a0-150-g660b845a87

**Comment by NormalPerson7** [comment]
Just to confirm what happened here: you entered a level and got the "You hear the sound of rushing water" alert, but then subsequently never got any information about where it should be?

Portal vaults tend to expire 600-800 turns after you first enter the level, including any time spent off-level. Are you sure you weren't off the level for that amount of time before you searched?

**Comment by jejorda2** [comment]
Yes, I got a message upon entering the level. (Through a shaft, I think.) Then I walked the entire perimeter of the level, hoping to get another message telling me I was close at some point.
Then I found the portal entrance near the center of the level and entered it. Never got any of the location hint messages.

**Comment by CrawlOdds** [comment]
Closing as this is consistent with a bug fixed in 353cd70
